libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/internal/ui/LocationPuckComponent.kt

```diff
@@ -25,49 +17,12 @@ class LocationPuckComponent(
     override fun onAttached(mapboxNavigation: MapboxNavigation) {
         super.onAttached(mapboxNavigation)
 
-        coroutineScope.launch {
-            locationProvider.waitForFirstLocation()
-            map.getStyle {
```

Comment:
Why is it not needed anymore? And waiting for the first location.

Reply:
Waiting for the first location was added in #5555. I think it was an issue with `getStyle`, which is now also removed. We don't do any of that in 1TAP and never had any issues with location puck.